### DISCUSSION TITLE
What's inside the Docker image amazon-k8s-cni:v1.11.3-eksbuild.1?

### DISCUSSION THREAD

**Discussion by suzaku** [aws/amazon-vpc-cni-k8s#2134] [Q&A] [answered]
I can only find the tag v1.11.3 in this repo.
Is `v1.11.3-eksbuild.1` built using version v1.11.3 of this project?

**Comment by jayanthvn** [comment, marked as answer]
Yes v1.11.3-eksbuild.1 and v1.11.3 are same images. We add 2 tags and the image with eksbuild.1 is vendored through managed/default addons for EKS cluster while the tag v1.11.3 is released on GitHub.

**Reply by suzaku** [reply]
Thanks.
